executable/Current.re

```diff
@@ -0,0 +1,11 @@
+open Fnm;
+
+let run = () => {
+  let%lwt currentVersion = Versions.getCurrentVersion();
+  switch (currentVersion) {
+  | Some({name}) => Console.log(name)
+  | _ => Console.log("none")
+  };
```

Comment:
So slick

Reply:
I hope so, this is the first thing I'm doing in OCaml/Reason, ignoring obligatory Hello World, and definitely isn't much more advanced 😁